README.md

```diff
@@ -178,33 +178,41 @@ To enable this configuration use the `extends` property in your
+| [`testing-library/no-await-sync-events`](./docs/rules/no-await-sync-events.md)                       | Disallow unnecessary `await` for sync events                                         |     |                                                                   |
+| [`testing-library/no-await-sync-query`](./docs/rules/no-await-sync-query.md)                         | Disallow unnecessary `await` for sync queries                                        |     | ![dom-badge][] ![angular-badge][] ![react-badge][] ![vue-badge][] |
+| [`testing-library/no-container`](./docs/rules/no-container.md)                                       | Disallow the use of container methods                                                |     | ![angular-badge][] ![react-badge][] ![vue-badge][]                |
+| [`testing-library/no-debug`](./docs/rules/no-debug.md)                                               | Disallow the use of debugging utilities like `debug`                                 |     | ![angular-badge][] ![react-badge][] ![vue-badge][]                |
+| [`testing-library/no-dom-import`](./docs/rules/no-dom-import.md)                                     | Disallow importing from DOM Testing Library                                          | 🔧  | ![angular-badge][] ![react-badge][] ![vue-badge][]                |
+| [`testing-library/no-manual-cleanup`](./docs/rules/no-manual-cleanup.md)                             | Disallow the use of `cleanup`                                                        |     |                                                                   |
+| [`testing-library/no-node-access`](./docs/rules/no-node-access.md)                                   | Disallow direct Node access                                                          |     | ![angular-badge][] ![react-badge][] ![vue-badge][]                |
+| [`testing-library/no-promise-in-fire-event`](./docs/rules/no-promise-in-fire-event.md)               | Disallow the use of promises passed to a `fireEvent` method                          |     | ![dom-badge][] ![angular-badge][] ![react-badge][] ![vue-badge][] |
+| [`testing-library/no-render-in-setup`](./docs/rules/no-render-in-setup.md)                           | Disallow the use of `render` in testing frameworks setup functions                   |     |                                                                   |
+| [`testing-library/no-unnecessary-act`](./docs/rules/no-unnecessary-act.md)                           | Disallow wrapping Testing Library utils or empty callbacks in `act`                  |     |                                                                   |
+| [`testing-library/no-wait-for-empty-callback`](./docs/rules/no-wait-for-empty-callback.md)           | It's preferred to avoid empty callbacks in `waitFor` and `waitForElementToBeRemoved` |     | ![dom-badge][] ![angular-badge][] ![react-badge][] ![vue-badge][] |
```

Comment:
Description was
> Disallow empty callbacks for `waitFor` and `waitForElementToBeRemoved`

Reply:
I think the previous one is better, can you change it in the rule description?